frontend/src/components/explorer/mapViewer/SidebarDataPanels.vue

```diff
@@ -264,6 +264,17 @@ export default {
       return l.join('');
     },
   },
+  watch: {
+    selectionData(newData, oldData) {
+      if (
+        (newData.data && !oldData.data)
+        || (!newData.data && oldData.data)
+        || (newData.data && oldData.data && newData.data.id !== oldData.data.id)
+      ) {
```

Comment:
Hmm I don't intuitively understand, so I'm going to ask - why wouldn't this work?
```
      if (newData.data) {
```

Reply:
Hmm, this is a pattern I picked up previously to compare the ids and I didn't give this too much thought. I tested it out now and indeed it looks like that it is not necessary. Actually it looks like the checking could be removed completely. Fixed in: https://github.com/MetabolicAtlas/MetabolicAtlas/pull/628/commits/349a9bf0cb0cf0284313e0130966f58df1843997